test/TestUtilities/Test.Utility/PackageManagement/TestSolutionManager.cs

```diff
@@ -17,19 +17,43 @@ namespace Test.Utility
 {
     public class TestSolutionManager : ISolutionManager, IDisposable
     {
+        private static readonly string GLOBAL_PACKAGES_ENV_KEY = "NUGET_PACKAGES";
+
         public List<NuGetProject> NuGetProjects { get; set; } = new List<NuGetProject>();
+
         public INuGetProjectContext NuGetProjectContext { get; set; } = new TestNuGetProjectContext();
 
+        public string NuGetConfigPath { get; set; }
+
+        public string GlobalPackagesFolder { get; set; }
+
+        public string PackagesFolder { get; set; }
+
         public string SolutionDirectory { get; }
 
-        private const string PackagesFolder = "packages";
 
         private TestDirectory _testDirectory;
 
+        private readonly string _configContent = @"<?xml version='1.0' encoding='utf-8'?>
+<configuration>
+  <packageSources>
```

Comment:
dont add sources, just add GlobalPackagesFolder.

Reply:
We need source to get packages that do not exist on disk.